system-tests/lib/system-tests.ts

```diff
@@ -342,7 +342,7 @@ const isVideoSnapshotError = (err: Error) => {
   _.pull(added, sometimesAddedVideoSnapshotLine, sometimesAddedSpacingLine)
   _.pull(deleted, sometimesDeletedVideoSnapshotLine, sometimesAddedSpacingLine)
 
-  return _.isEqual(added, expectedAddedVideoSnapshotLines) && _.isEqual(deleted, expectedDeletedVideoSnapshotLines)
+  return _.isEqual(added, expectedAddedVideoSnapshotLines) && (deleted.length === 0 || _.isEqual(deleted, expectedDeletedVideoSnapshotLines))
```

Comment:
This is modifying an existing workaround for the firefox video error... previously when video was being compressed there were lines that were missing from the saved snapshot... since with compression off, the video name isn't logged, now there are not deleted lines.